README.md

```diff
@@ -59,17 +59,21 @@ provider "aws" {
   region = "us-west-2"
 
   # Currently our modules require the older v2 AWS provider, as upgrading to v3 has notable breaking changes.
```

Comment:
Looks like this comment is not valid anymore?

Reply:
yes